src/libraries/System.Speech/src/System.Speech.csproj

```diff
@@ -7,6 +7,8 @@
     <NoWarn>$(NoWarn);CS0649;SA1129;IDE0059;IDE0060;CA1822;CA1852</NoWarn>
     <Nullable>annotations</Nullable>
     <IsTrimmable>false</IsTrimmable>
+    <!-- Public API not fully documented. -->
+    <NoWarn>$(NoWarn);1591</NoWarn>
```

Comment:
FYI @dotnet/area-system-speech area owners: This assembly is missing public documentation.

Reply:
https://github.com/dotnet/runtime/issues/87711